app/src/main/java/com/x8bit/bitwarden/data/autofill/builder/FilledDataBuilderImpl.kt

```diff
@@ -76,7 +84,7 @@ class FilledDataBuilderImpl(
             ?.getOrLastOrNull(inlineSuggestionsAdded)
 
         return FilledData(
-            filledPartitions = filledPartitions,
+            filledPartitions = filledPartitions.take(n = MAX_PARTITIONS_COUNT),
```

Comment:
Should we really be limiting just the number of partitions? I thought typically we are dealing with just a single partition that has potentially a large number or suggestions inside it.

Reply:
OK I think this is just a terminology issue; `AutofillPartition` is what you'd typically have one of and `FilledPartition` is what contains the possible cipher information that could fill it. So what you're doing does make sense.